README.md

```diff
@@ -1 +1,35 @@
-# Capstone1
\ No newline at end of file
+# Youth Week
+
+> a website about the International youth Week event. for the microverse capstone1.
+
+## Built With
+
+- HTML
+- SCSS
+- JavaScript
+
+## Live Project
+
+[Live Link](http://127.0.0.1:5500/index.html)
+
+## Video Presentation
+
+[Loom Link](https://www.loom.com/share/74a5724e4b0e46b299357fd4fd807371/)
+
+## Author
+
+1. Name: Adekunle Akinkunmi Ismail
+2. Github: @<https://github.com/ismailakinkunmi>
+3. LinkedIn: @<https://www.linkedin.com/in/ismail-akinkunmi-adekunle-b810aa155/>
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
```

Comment:
- [ ] [REQUIRED] As stated in the project requirements
The [Creative Commons license](https://www.behance.net/adagio07) of the design requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project. Please fix that and submit it for review.😊